semantic_release_template.yaml

```diff
@@ -0,0 +1,12 @@
+name: screwdriver-cd/semantic-release
+version: '1.0'
+description: Template for semantic release
+maintainer: idk
+config:
+  image: node:6
+  steps:
+    - check_prerequisite: |
+        if [ -z "$NPM_TOKEN" ]; then echo "NPM Token not set to NPM_TOKEN. Exiting..." && exit 1; fi
+        if [ -z "$GH_TOKEN" ]; then echo "Github Token not set to GH_TOKEN. Exiting..." && exit 1; fi
+    - install_binaries: npm i -g semantic-release@^7
```

Comment:
If we install globally, this won't work..
https://github.com/screwdriver-cd/data-schema/blob/master/package.json#L36

Reply:
Your link is to the `package.json` version. I'm not sure what operation you're trying to point out that overrides the default verification.

Reply:
@FenrirUnbound  it's this one https://github.com/screwdriver-cd/data-schema/blob/master/package.json#L36

Reply:
LOL @FenrirUnbound should be https://github.com/screwdriver-cd/data-schema/blob/master/package.json#L36